static/js/src/advantage/react/components/Subscriptions/SubscriptionDetails/SubscriptionDetails.tsx

```diff
@@ -251,27 +247,6 @@ export const SubscriptionDetails = forwardRef<HTMLDivElement, Props>(
                     />
                   </Portal>
                 )}
-                {(subscription.statuses.is_expired ||
-                  subscription.statuses.is_in_grace_period) &&
-                  (subscription.type == "monthly" ||
-                    subscription.type == "yearly") && (
-                    <Button
-                      appearance="neutral"
-                      className="p-subscriptions__details-action"
-                      data-test="renew-button"
-                      disabled={editing}
-                      onClick={() => {
-                        setShowingReBuyExpiredModal(true);
-                        sendAnalyticsEvent({
-                          eventCategory: "Advantage",
-                          eventAction: "subscription-rebuy-expired-modal",
-                          eventLabel: "subscription rebuy expired modal opened",
-                        });
-                      }}
-                    >
-                      Renew subscription&hellip;
-                    </Button>
-                  )}
```

Comment:
I don't think we need to remove this. In fact, it's probably a good idea to keep it for the future. It will certainly work during the grace period, which should cover most customer's use cases.

Reply:
I'll re-add in a second.

Reply:
OK, should be good to go now.